ideas/000-methrics/README.md

```diff
@@ -0,0 +1,164 @@
+---
+id: 000-methrics
+title: Methrics
+status: draft
+created: 2018-05-17
+category: research
+lead-contributor:
+contributors:
+    -
+    -
+    -
+exit-criteria: no
+success-metrics: no
+clear-roles: no
+future-iteration: no
+roles-needed:
+    - QA
+    - PM
+    - UXR
+    - Clojure dev
+    - Go dev
+    - Contracts dev
+    - Designer
+    - Community
+---
+
+## Preamble
+
+    Idea: 000-methrics
+    Title: Methrics
+    Status: Draft
+    Created: 2018-05-17
+    Requires:
+    Replaces:
+
+## Summary
+
+A decentralized and anonymized approach to application metrics
+
+## Swarm Participants
+
+TODO
+
+- Lead Contributor:
+- Evaluator (defaults to lead contributor):
+- Contributor:
+- Contributor:
+- QA:
+- PM (required for user-facing):
+- UXD/R (required for user-facing swarms): @patrick, @hester
+
+## Product Overview
+
+Striking the balance between delivering a beautiful app and respecting the privacy of the user remains a task fraught with difficult tradeoffs.
+
+One such tradeoff is the collection of application metrics from users as they use the application - in that process, people have little insight into what is being collected and how it may be used, within or outside of its intended purpose.
+
+The decentralized open metrics idea is an experiment to explore fully transparent metrics collection where the data is available to the user and to anyone else as it's being collected, with minimal leakage outside of its stated scope.
+
+Metrics are generally not limited to data that is explicitly generated for the purpose of tracking, but also include other traces that the user might leave behind as they navigate the application, such as transactions and dapp assets hosted on web sites, and part of the product idea is to explore ways to expose and break down these trails for the user such that they become accessible and understandable.
+
+To explore as part of this project is how to ensure that data retention laws such as GDPR are accounted for.
+
+### User Stories
+
+* I want to understand what trails I leave behind when using the application, including blockchain transactions and dapp interactions, and how these may be combined
+* I want to contribute to the development of Status anonymously by letting the application collect tracking data
+* I want to limit specific types of data that Status collects when I use the application
+* I want to export the data conveniently for my own analysis
+* I want meaningful information that I can make sense of, opposed to datapoints
+
```

Comment:
This is currently user centric, which is fine. But we should also make it clear what this data is being used for by Status. "I want to build the right features for our users, and want to understand adoption"

Reply:
done